content/2021/2021-04/2021-04-12-pulsae-weekly.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "2021-04-12-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2021-04-05 ~ 2021-04-11"
+date: 2021-04-05 ~ 2021-04-11
+description: "This is the Pulsar community weekly update for 2021-04-05 ~ 2021-04-11, with updates on Pulsar client, broker, transactions, and so on."
+id: "2021-04-12-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2021-04-05 ~ 2021-04-11
+
+This is the Pulsar community weekly update for 2021-04-05 ~ 2021-04-11, with updates on Pulsar client, broker, transactions, and so on.
+
+### Pulsar Highlight
+
+[Schema Registry] Introduce a new high-level `GenericObject` interface that represents every logic value on the Pulsar topic. It supports writing sinks that work with every Schema type.
+
+by (@[eolivelli](https://github.com/eolivelli))
+
+### Development
+
+- [PIP-82] [Broker] Add a task to publish resource usage to a topic.
+
+    https://github.com/apache/pulsar/pull/10008 (@[bharanic-dev](https://github.com/bharanic-dev))
+
+### Notable Feature
+
+- [Test] Refactor existing Pulsar Function unit tests.
+
+    https://github.com/apache/pulsar/pull/10125 (@[david-streamlio](https://github.com/david-streamlio))
+
+- [C++] Support configuring the memory limit from the C++ API.
+
+    https://github.com/apache/pulsar/pull/10145 (@[merlimat](https://github.com/merlimat))
+
+- [Functions] Expose Prometheus metrics for Pulsar functions in the local-run mode.
```

Comment:
I think we could just use `local run` instead of `local-run`, see https://pulsar.apache.org/docs/en/functions-deploy/#local-run-mode